intel/docs/Grafana-Dashboard.md

```diff
@@ -0,0 +1,63 @@
+# Intel Distribution of Istio (Grafana Dashboard)
+Prerequisites:
+* Kubernetes cluster with at least one Ice Lake node
+* Cert-Manager
+* Istioctl
+* Helm
+
+This README contains:
+- [Intel Distribution of Istio Grafana Dashboard](#deployment)
+- [Removal of telemetry deployment](#uninstallation)
```

Comment:
Just say:

This README contains: 
Installing Grafana dashboard
Uninstallation